<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UsuariosForm</class>
 <widget class="QWidget" name="UsuariosForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>532</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QPushButton" name="nuevo_pushButton">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Nuevo</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/Icons.qrc">
           <normaloff>:/Icons/1349716393_monotone_add_plus.png</normaloff>:/Icons/1349716393_monotone_add_plus.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+N</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="editar_pushButton">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Editar</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/Icons.qrc">
           <normaloff>:/Icons/1349716425_doc_edit.png</normaloff>:/Icons/1349716425_doc_edit.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+E</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="pB_buscar">
         <property name="text">
          <string>Buscar</string>
         </property>
         <property name="icon">
          <iconset resource="../Icons/Icons.qrc">
           <normaloff>:/Icons/1349716430_zoom.png</normaloff>:/Icons/1349716430_zoom.png</iconset>
         </property>
         <property name="iconSize">
          <size>
           <width>24</width>
           <height>24</height>
          </size>
         </property>
         <property name="shortcut">
          <string>Ctrl+B</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTabWidget" name="Module_tabWidget">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <widget class="QWidget" name="list_tab">
        <attribute name="title">
         <string>Lista</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="0" column="0">
          <widget class="QTableView" name="list_tableView">
           <property name="mouseTracking">
            <bool>true</bool>
           </property>
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="alternatingRowColors">
            <bool>true</bool>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::SolidLine</enum>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="detail_tab">
        <attribute name="title">
         <string>Detalle</string>
        </attribute>
        <layout class="QGridLayout" name="gridLayout_4">
         <item row="0" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="0" column="1">
              <widget class="QLineEdit" name="le_pApellido"/>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_16">
               <property name="text">
                <string>Número de Documento</string>
               </property>
              </widget>
             </item>
             <item row="5" column="2">
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Teléfono Fam.</string>
               </property>
              </widget>
             </item>
             <item row="10" column="0" colspan="4">
              <widget class="QPlainTextEdit" name="tE_obs">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="minimumSize">
                <size>
                 <width>0</width>
                 <height>0</height>
                </size>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>50</height>
                </size>
               </property>
              </widget>
             </item>
             <item row="7" column="1">
              <widget class="QLineEdit" name="le_Usuario">
               <property name="readOnly">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="5" column="0">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>Teléfono</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Dirección</string>
               </property>
              </widget>
             </item>
             <item row="6" column="2">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Tipo Colaborador</string>
               </property>
              </widget>
             </item>
             <item row="4" column="2">
              <widget class="QLabel" name="label_27">
               <property name="text">
                <string>Celular</string>
               </property>
              </widget>
             </item>
             <item row="4" column="0">
              <widget class="QLabel" name="label_28">
               <property name="text">
                <string>Fecha de Nacimiento</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QComboBox" name="cB_tDoc"/>
             </item>
             <item row="9" column="2" colspan="2">
              <spacer name="horizontalSpacer_3">
               <property name="orientation">
                <enum>Qt::Horizontal</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>40</width>
                 <height>20</height>
                </size>
               </property>
              </spacer>
             </item>
             <item row="2" column="3">
              <widget class="QLineEdit" name="le_numDoc"/>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Primer Apellido</string>
               </property>
              </widget>
             </item>
             <item row="0" column="2">
              <widget class="QLabel" name="label_29">
               <property name="text">
                <string>Segundo Apellido</string>
               </property>
              </widget>
             </item>
             <item row="5" column="1">
              <widget class="QLineEdit" name="le_telf"/>
             </item>
             <item row="2" column="0">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Tipo de Documento</string>
               </property>
              </widget>
             </item>
             <item row="1" column="3">
              <widget class="QComboBox" name="cb_sexo">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="7" column="2">
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Cambiar Contraseña:</string>
               </property>
              </widget>
             </item>
             <item row="7" column="0">
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Nombre de Usuario</string>
               </property>
              </widget>
             </item>
             <item row="4" column="1">
              <widget class="QDateEdit" name="de_fechaNac">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="displayFormat">
                <string>dd/MM/yyyy</string>
               </property>
               <property name="calendarPopup">
                <bool>true</bool>
               </property>
              </widget>
             </item>
             <item row="0" column="3">
              <widget class="QLineEdit" name="le_sApellido"/>
             </item>
             <item row="3" column="1" colspan="3">
              <widget class="QLineEdit" name="le_direccion"/>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>Nombres</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_17">
               <property name="text">
                <string>Sexo:</string>
               </property>
              </widget>
             </item>
             <item row="9" column="0">
              <widget class="QLabel" name="label_30">
               <property name="text">
                <string>Observaciones</string>
               </property>
              </widget>
             </item>
             <item row="6" column="1">
              <widget class="QLineEdit" name="le_email"/>
             </item>
             <item row="4" column="3">
              <widget class="QLineEdit" name="le_cell"/>
             </item>
             <item row="1" column="1">
              <widget class="QLineEdit" name="le_nombre"/>
             </item>
             <item row="5" column="3">
              <widget class="QLineEdit" name="lE_telfFam"/>
             </item>
             <item row="6" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Email :</string>
               </property>
              </widget>
             </item>
             <item row="8" column="0">
              <widget class="QLabel" name="label_6">
               <property name="text">
                <string>Habilitado</string>
               </property>
              </widget>
             </item>
             <item row="8" column="1">
              <widget class="QComboBox" name="cB_habilitado">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
              </widget>
             </item>
             <item row="7" column="3">
              <widget class="QComboBox" name="cB_cambiarpass">
               <property name="editable">
                <bool>false</bool>
               </property>
              </widget>
             </item>
             <item row="6" column="3">
              <widget class="QComboBox" name="cB_tipoCol"/>
             </item>
             <item row="8" column="2" colspan="2">
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Contraseña: por defecto es 1234</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QTableView" name="tableView_permisos">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>300</width>
               <height>0</height>
              </size>
             </property>
             <property name="editTriggers">
              <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed|QAbstractItemView::SelectedClicked</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="0">
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <spacer name="horizontalSpacer_4">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="save_pushButton">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Guardar</string>
             </property>
             <property name="icon">
              <iconset resource="../Icons/Icons.qrc">
               <normaloff>:/Icons/1349716356_save.png</normaloff>:/Icons/1349716356_save.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Ctrl+G</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="cancel_pushButton">
             <property name="font">
              <font>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>Cancelar</string>
             </property>
             <property name="icon">
              <iconset resource="../Icons/Icons.qrc">
               <normaloff>:/Icons/1349716376_monotone_close_exit_delete.png</normaloff>:/Icons/1349716376_monotone_close_exit_delete.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>24</width>
               <height>24</height>
              </size>
             </property>
             <property name="shortcut">
              <string>Ctrl+U</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_5">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>le_pApellido</tabstop>
  <tabstop>le_sApellido</tabstop>
  <tabstop>le_nombre</tabstop>
  <tabstop>cb_sexo</tabstop>
  <tabstop>cB_tDoc</tabstop>
  <tabstop>le_numDoc</tabstop>
  <tabstop>le_direccion</tabstop>
  <tabstop>de_fechaNac</tabstop>
  <tabstop>le_cell</tabstop>
  <tabstop>le_telf</tabstop>
  <tabstop>lE_telfFam</tabstop>
  <tabstop>le_email</tabstop>
  <tabstop>cB_tipoCol</tabstop>
  <tabstop>le_Usuario</tabstop>
  <tabstop>cB_cambiarpass</tabstop>
  <tabstop>cB_habilitado</tabstop>
  <tabstop>tE_obs</tabstop>
  <tabstop>tableView_permisos</tabstop>
  <tabstop>save_pushButton</tabstop>
  <tabstop>cancel_pushButton</tabstop>
  <tabstop>Module_tabWidget</tabstop>
  <tabstop>list_tableView</tabstop>
  <tabstop>editar_pushButton</tabstop>
  <tabstop>nuevo_pushButton</tabstop>
 </tabstops>
 <resources>
  <include location="../Icons/Icons.qrc"/>
 </resources>
 <connections/>
</ui>
